docs/sources/tempo/getting-started/metrics-from-traces.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Metrics from traces
+description: Learn about how we can correlate traces and metrics.
+weight: 700
+---
+
+# Metrics from traces
+
+RED metrics can be used to drive service graphs and other ready-to-go visualizations of your span data. RED metrics represent:
+
+- Rate, the number of requests per second
+- Errors, the number of those requests that are failing
+- Duration, the amount of time those requests take
+
+For more information about RED method, refer to [The RED Method: How to instrument your services](/blog/2018/08/02/the-red-method-how-to-instrument-your-services/).
+
+## Use-cases for span metrics
+{{% admonition type="note" %}}
+Metrics generation is disabled by default. Contact Grafana Support to enable metrics generation in your organization.
+{{% /admonition %}}
+
```

Comment:
I think we also need a bit more of an intro or segue here.

Reply:
I pushed up a small introduction. I'm borrowing some of the text from the metrics-generator page here. What do you think?

Reply:
Looks good. It gives just enough of an introduction to the section. We can always iterate and refine later, too.